shared/src/ledger/ibc/storage.rs

```diff
@@ -155,54 +167,54 @@ pub fn capability_index_key() -> Key {
 #[cfg(any(feature = "ibc-vp", feature = "ibc-vp-abci"))]
```

Comment:
(Unrelated to this PR) I didn't realize, but I think we can probably remove all the `#[cfg(any(feature = "ibc-vp", feature = "ibc-vp-abci"))]` annotations, as they will be always enabled anyway

Reply:
I tried that before and wasm build failed. Some functions depend on `ibc-rs`.
After `ibc-rs` supports `no-std`, we can remove all.
Or, we can avoid annotating each function by splitting the `storage` file.